docs/reference/2-ods-api/getting-started/source-code-installation/readme.md

```diff
@@ -0,0 +1,718 @@
+
+:::
+
+```powershell
+PS C:\Ed-Fi-ODS-Implementation> initdev
+
+----------------------------------------
+    Invoke-NewDevelopmentAppSettings
+----------------------------------------
+
+<trimmed output...>
+```
+
+This command creates databases, generates code templates, and compiles projects
+in the solution. Some considerations while running the script:
+
+* The `initdev` script may not finish with a command prompt when it is
+  automatically loaded with the solution in Visual Studio in some circumstances.
+  Simply press **Enter**, and ignore any messages that appear in the console
+  window.
+* Initializing the development environment will take several minutes to
+  complete. Some developers report encountering an error during the
+  initialization process the first time it is run. This is due to an
+  intermittent timing issue. The issue generally resolves itself when
+  the `initdev` process is run a second time.
+
+A successful `initdev` execution will display the tasks executed and their
+duration as shown below:
+
+```powershell
+<trimmed output...>
+
+Duration Task
+-------- ----
+00:01.92 Invoke-NewDevelopmentAppSettings
+00:02.69 Install-Plugins
+00:01.57 Install-CodeGenUtility
+00:08.19 Invoke-CodeGen
+00:29.14 Invoke-RebuildSolution
+00:01.48 Install-DbDeploy
+00:15.78 Reset-TestAdminDatabase
+00:26.09 Reset-TestSecurityDatabase
+00:20.62 Reset-TestPopulatedTemplateDatabase
+00:14.31 Reset-AdminDatabase
+00:25.81 Reset-SecurityDatabase
+00:00.78 Remove-SandboxDatabases
+00:19.23 Reset-MinimalTemplateDatabase
+00:20.55 Reset-PopulatedTemplateDatabase
+-        -
+03:11.35 Initialize-DevelopmentEnvironment
+```
+
+## Step 5. Build the Visual Studio Solution
+
+To build the solution from within Visual Studio:
+
+1. Start Visual Studio. Within Visual Studio, open the "Ed-Fi-Ods.sln" solution
+    file from the C:\\Ed-Fi-ODS-Implementation\\Application directory.
+2. Select **Build** > **Build Solution** (or press **Ctrl**+**Shift**+**B**).
+
+<details>
+<summary>View detail and options...</summary>
+
+## Code Generation During Build
+
+The following diagram shows how MetaEd generated artifacts are used to create
+the API for the Ed-Fi ODS using code generators within the solution. Code
+generation uses Api Model JSON file to understand the structure that it uses to
+generate data access code. Code generation also depends on the
+DatabaseViews.generated.json which is generated one time using the
+"EdFi\_Ods\_Empty" database and subsequently provided by the source code
+repository.
+
+```mermaid
+flowchart LR
+    model@{ shape: doc, label: "APIModel.json" }
+    metaed[MetaEd] --> model
+
+    db[(EdFi_Ods_Empty)]
+
+    metaed --> db
+    views@{ shape: doc, label: "Database Views" }
+    db -.-> views
+
+    gen@{ shape: processes, label: "Code Generators" }
+
+    model --> gen
+    views --> gen
+
+    gen --> generated["compiled C#, json, XML"]
+
+    generated --> server["Ed-FI ODS/API service"]
+```
```

Comment:
```suggestion
```mermaid
flowchart LR
    model["APIModel.json"]
    metaed[MetaEd] --> model

    db[(EdFi_Ods_Empty)]

    metaed --> db
    views["Database Views"]
    db -.-> views

    gen[["Code Generators"]]

    model --> gen
    views --> gen

    gen --> generated["compiled C#, json, XML"]

    generated --> server["Ed-FI ODS/API service"]
```
```

Reply:
Seems like Docusaurus does not properly support the @directives for formatting the shapes and is causing a syntax error.